### ISSUE TITLE
backuppc 4.0.0alpha3 with rsync-bpc 3.0.9.3 and 3.0.9.4 - error in rsync protocol data stream

### ISSUE THREAD

**Issue by pavelbenisek** [backuppc/rsync-bpc#1]
Hi I am using backuppc 4.0.0.alpha3 with rsync-bpc 3.0.9.3 and 3.0.9.4.
in both cases I keep getting the following error when trying to do full backup of hosts with 570 000 files.
```
2017-01-05 14:50:00 full backup started for directory /
2017-01-05 16:41:54 Got fatal error during xfer (rsync error: error in rsync protocol data stream (code 12) at io.c(629) [generator=3.0.9.4])
2017-01-05 16:41:59 Backup aborted (rsync error: error in rsync protocol data stream (code 12) at io.c(629) [generator=3.0.9.4])
```
Initially I was continuing on top of the old v3 backups, but then I removed them hoping the problem lies somewhere between the v3 vs. v4 incompatibility. Unfortunately it did not help and I keep getting the same error somewhere around 180 000 files being processed.

**Comment by craigbarratt** [comment]
Can you repeat the test with the latest rsync_bpc from git?  It's easy to build and install (or you can just point $Conf{RsyncBackupPCPath} at the new rsync_bpc without installing).  There have been several bug fixes since 3.0.9.4.

**Comment by pavelbenisek** [comment]
After managing (which took me while) to install it again, this issue went away.
The only necessary action was to compile the rsync_bpc from sources - master branch.
Thanks craigbarrat.
